src/Keymapping-Core/KMShortcutPrinter.class.st

```diff
@@ -48,6 +48,22 @@ KMShortcutPrinter class >> toString: aShortcut [
 		visit
 ]
 
+{ #category : #visiting }
+KMShortcutPrinter >> genericSpecialCharacter: char [ 
+
+	char = Character cr ifTrue: [ ^ 'Enter' ].
```

Comment:
Shouldn't we manage lf also? In case in the future the line return of Pharo change?

Reply:
nope. 
this is not "return line" but "return the char"